packages/ember-mirage/src/test-support/setup-mirage.js

```diff
@@ -11,7 +11,7 @@ import startMirage from './start-mirage';
  NOTE: the `hooks = self` is for mocha support
  @hide
  */
-export default function setupMirage(hooks = self, { makeServer, ...options }) {
```

Comment:
Im fine removing default, but since this is the only thing exported from this file, shouldnt it be the default?

Reply:
no one can import from the file directly it's a private file :partying_face: